crowbar_framework/app/assets/javascripts/barclamps/neutron/application.js

```diff
@@ -299,6 +306,7 @@ function ml2_mechanism_drivers_check() {
     $('#ml2_type_drivers_container').hide();
     $('#ml2_type_drivers_default_provider_network_container').hide();
     $('#ml2_type_drivers_default_tenant_network_container').hide();
+    $('#vxlan_group_container').show();
```

Comment:
Isn't this backwards? Shouldn't we hide the vxlan options for now when linuxbridge is selected?